.github/workflows/release.yml

```
@@ -0,0 +1,23 @@
+---
+name: "Release new version on Ansible Galaxy"
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+
+  release:
+    name: "Release new version on Ansible Galaxy"
+    runs-on: "ubuntu-22.04"
+    if: "startsWith(github.ref, 'refs/tags/v')"
+    needs: ["lint", "test"]
```

Sorry, I missed to spot this line in the first review.

```suggestion
    needs: ["ansible-lint", "test"]
```

---

This probably won't work, because we cannot reference jobs from other workflows.
I have changed the workflow so that it only runs when a new release is published. Of course, we then have to make sure ourselves that all the necessary workflows for the corresponding git tag have been passed.